Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -862,14 +862,14 @@ class BrowserViewController: UIViewController {
             return
         }
 
-        if webView == tabManager.selectedTab?.webView, let tab = tabManager.selectedTab, let helper = tab.getContentScript(name: ContextMenuHelper.name()) as? ContextMenuHelper {
+        if let helper = tab.getContentScript(name: ContextMenuHelper.name()) as? ContextMenuHelper {
```

Comment:
Checking if this tab is selected is not necessary, and probably better not to check that in case the tab is de-selected but we still need to install the gesture handler.